packages/paste-website/src/pages/patterns/copy-to-clipboard/index.mdx

```diff
@@ -0,0 +1,454 @@
+---
+title: Copy to clipboard
+description: Give customers guidance on what steps to take if content is either unavailable or can't be displayed.
+slug: /patterns/copy-to-clipboard/
```

Comment:
PS: this file and slug should be under the `library` folder/path

Reply:
wait isn't this file auto-generated? can i modify this `.mdx` file?

Reply:
never mind, i think i did add this manually and forgot